Add Czech locale

tukusejssirs opened the pull request:
- fixes #125.

tukusejssirs commented:
Note that this PR (when finished) should be merged _after_ #213.

For now, I used only resources from [igneus/calendarium-romanum](https://github.com/igneus/calendarium-romanum), specifically from these files:

- [cs.yml](https://github.com/igneus/calendarium-romanum/blob/master/config/locales/cs.yml);
- [czech-cs.txt](https://github.com/igneus/calendarium-romanum/blob/master/data/czech-cs.txt).

The current (initial) version covers all keys used in `general.ts` and (current, invalid) `czechRepublic.ts`.

---

My TODO:
- [ ] localise all strings;
- [ ] sort the keys in all sections of the locale;
- [ ] run `prettier`.

tukusejssirs commented:
Until now, 50 % of `sactorale` keys are localised. Everything outside `sanctorale` is localised. As I did with Latin locale, the progress can be found in individual commit message description (the latest commit message contains `355 keys done of 701 (50 %)`).

There is (and will be) great list of TODOs (because I am not a native Czech speaker). I hope @igneus will find some time to review this PR. As I think the review will take some time, it might be a good idea to start it now (the sooner, the better), however, feel free to do it or reject doing it. :smiley:

Some notes:
- I localise all (missing) keys from top to bottom; I don’t skip a key (last line localised is currently L206).
- Keys that are not localised yet are commented out.
- As I started with the strings I found in `calendarium-romanum`, some keys are skipped, but only after L206.
- I placed my notes (and TODOs) after the key they are related to.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    christmas: 'Narození Páně',
+    christ_the_king_sunday: 'Ježíše Krista krále',
+    corpus_christi: 'Těla a krve Páně',
+    divine_mercy_sunday: 'Neděle Božího milosrdenstv',
```
typo (missing last letter)

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    trinity_sunday: 'Nejsvětější Trojice',
+  },
+  sanctoral: {
+    '205_blessed_martyrs_of_japan': '205 blahoslavených mučedníků Japonska',
```
My personal preference would be `... japonských mučedníků`, possibly even `Bl. 205 japonských mučedníků`

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    aloysius_stepinac_bishop: 'Bl. Aloise Stepinace, biskupa a mučedníka',
+    andre_grasset_priest: 'Bl. Ondřeje Grasset, kněze a mučedníka',  // TODO: Check the surname inflection
+    angela_salawa_virgin: 'Bl. Anděly Salawové, panny',
+    anthony_julian_nowowiejski_bishop_and_companions_martyrs: 'Bl. Antonína Juliana Nowowiejski, biskupa, a druhů, mučedníků',  // TODO: Check the surname inflection
```
Polish names like this are usually inflected. Also the name "Julian" would usually be "bohemized":
`Bl. Antonína Juliána Nowowiejského, biskupa, a druhů, mučedníků`

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    andre_grasset_priest: 'Bl. Ondřeje Grasset, kněze a mučedníka',  // TODO: Check the surname inflection
+    angela_salawa_virgin: 'Bl. Anděly Salawové, panny',
+    anthony_julian_nowowiejski_bishop_and_companions_martyrs: 'Bl. Antonína Juliana Nowowiejski, biskupa, a druhů, mučedníků',  // TODO: Check the surname inflection
+    augustine_kazotic_bishop: 'Bl. Augustina Kažotić, biskupa a mučedníka',
```
Croatian surname like this would also get inflected `Bl. Augustina Kažotiće, biskupa a mučedníka`

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    angela_salawa_virgin: 'Bl. Anděly Salawové, panny',
+    anthony_julian_nowowiejski_bishop_and_companions_martyrs: 'Bl. Antonína Juliana Nowowiejski, biskupa, a druhů, mučedníků',  // TODO: Check the surname inflection
+    augustine_kazotic_bishop: 'Bl. Augustina Kažotić, biskupa a mučedníka',
+    bartholomew_of_the_martyrs_fernandes_bishop: 'Sv. Bartoloměje od Mučedníků Fernandes, biskupa',
```
This would usually also get inflected `Sv. Bartoloměje od Mučedníků Fernandese, biskupa`
(Yes, I know it was me who said that foreign names are by default not inflected, but...)

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    bogumilus_of_dobrow_bishop: 'Bl. Bohumila, biskupa',
+    boleslawa_mary_lament_virgin: 'Bl. Boleslavy Marie Lamentové, panny',  // TODO: Lament or Lamentové?
+    bronislava_of_poland_virgin: 'Bl. Bronislavy, panny',
+    bronislaw_markiewicz_priest: 'Bl. Bronislava Markiewicz, kněze',
```
`Bl. Bronislava Markiewicze, kněze`

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    zepherin_namuncura: 'Bl. Zeffirína Namuncurá',
+    ceslaus_of_poland_and_hyacinth_of_poland_priests: 'Bl. Česlava a sv. Hyacinta, kněží',
+    charles_spinola_priest: 'Bl. Karla Spinoly, kněze a mučedníka',
+    columba_marmion_priest: 'Bl. Columby Marmiona, kněze',  // Note: Although foreign surnames should not be inflected by rule, this form was found on the Internet (feel free to correct it if it is incorrect) → https://www.google.com/search?q=Bl.+"Columby"+Marmion%2C+kněze+site%3A.cz
```
it's 100% correct, you can drop the note

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    ceslaus_of_poland_priest: 'Bl. Česlava, kněze',
+    dina_belanger_virgin: 'Bl. Diny Bélangerové, panny',
+    dominic_of_the_mother_of_god_barberi_priest: 'Bl. Dominika od Matky Boží Barberi, kněze',
+    edmund_bojanowski: 'Bl. Edmunda Bojanowski',
```
`Bl. Edmunda Bojanowského`

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    elizabeth_hesselblad_religious: 'Sv. Alžběty Hesselbladové, řeholnice',
+    emilie_tavernier_gamelin_religious: 'Bl. Emilie Tavernier-Gamelin, řeholnice',  // TODO: On the Net, I didn't found her name with `-ová` suffix
+    frederic_janssoone_priest: 'Bl. Frederika Janssoone, kněze',
+    george_matulaitis_bishop: 'Bl. Jiřího Matulaitis, biskupa',
```
Probably `Bl. Jiřího Matulaitise, biskupa`

tukusejssirs replied:
Just to make sure: should it be `Jiři` or `Jiřího` in this particular case?

igneus replied:
`Jiřího` here - the archaic declension (genitive `sv. Jiří`) is only ever used in relation to _the_ St. George.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    gratia_of_cattaro_religious: 'Bl. Gracie z Kotoru, řeholníka',  // Note: It's a male blessed named `Grácia` (or `Gratia`/`Grazia`/`Gracia`); see https://www.opatstvibrno.cz/opatstvi/svaty-augustin/augustiniani-svati-a-blahoslaveni.html
+    hemming_of_turku_bishop: 'Bl. Hemminga z Turku, biskupa',
+    henry_suso_priest: 'Bl. Jindřicha Suso, kněze',
+    honorat_kozminski_priest: 'Bl. Honoráta Koźmiński, kněze',
```
`Bl. Honoráta Koźmińského, kněze`

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    honorat_kozminski_priest: 'Bl. Honoráta Koźmiński, kněze',
+    hroznata_of_bohemia_martyr: 'Bl. Hroznaty, mučedníka',
+    ignatius_de_azevedo_priest_and_companions_martyrs: 'Bl. Ignáce de Azevedo, kněze, a druhů, mučedníků',
+    innocent_xi_pope: 'Bl. Inocenta XI., papeže',
```
Inocen*ce*

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    ignatius_de_azevedo_priest_and_companions_martyrs: 'Bl. Ignáce de Azevedo, kněze, a druhů, mučedníků',
+    innocent_xi_pope: 'Bl. Inocenta XI., papeže',
+    irish_martyrs: 'Bl. irských mučedníků',
+    istvan_sandor_religious: 'Bl. Štěpána Šándora, řeholníka a mučedníků',
```
the title is strange - should it be "řeholníka a mučedníka", or "..., řeholníka, a druhů, mučedníků"?

tukusejssirs replied:
The former is correct. :smiley:

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    irish_martyrs: 'Bl. irských mučedníků',
+    istvan_sandor_religious: 'Bl. Štěpána Šándora, řeholníka a mučedníků',
+    ivan_merz: 'Bl. Ivana Merze',  // Note: Surname is inflected in [here](https://www.bip.cz/cs/novinky/2018-08-blahoslaveny-splzenskymi-koreny-ivan-merz-a-jeho-cesta-ke-slunci)
+    jacinta_marto_and_francisco_marto: 'Sv. Hyacinty a Františka Martovi',
```
"Martových", eventually (not inflected) "Marto"

tukusejssirs replied:
Not that the source is 100 % correct, but I’ve got it from [catholica.cz](http://catholica.cz/?id=581).

> "Martových", eventually (not inflected) "Marto"

Now I am not sure what you suggest: should I use `Martových` or `Marto`? What do you think is (more) correct? Choose one please. :wink:

igneus replied:
The heading at catholica.cz is in nominative, "Martových" is correct genitive from this nominative.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    james_strzemie_bishop: 'Bl. Jakuba Strzemię, biskupa',
+    joan_of_portugal_virgin: 'Bl. Jany Portugalské, panny',
+    john_beyzym_priest: 'Bl. Jana Beyzyma, kněze',
+    john_henry_newman_priest: 'Sv. Jana Jindřicha Newmana, kněze',
```
This would look and sound strange (the way some 19th c. expressions sound strange nowadays) - usual is the form `Sv. Johna Henry Newmana, kněze`

tukusejssirs replied:
Okay, I see. But should it be `Henry` or rather `Henryho`?

igneus replied:
Both forms are used. I prefer the one with the second name undeclined (`Henry`) and perceive the other one as somewhat raw or less elegant, but I'm short of relevant arguments in favour of my preferred version.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    leonid_feodorov_priest: 'Bl. Leonida Fedorova, kněze a mučedníka',  // TODO: Is it `Fedorov` or `Fjodorov` in Czech? According to Google, `Fedorov` is used much more on Czech websites, however, it does not reflect the original (Russian) pronunciation
+    louis_zephirin_moreau_bishop: 'Bl. Ludvíka Zeffirína Moreau, biskupa',
+    marcel_callo_martyr: 'Bl. Marcela Callo, mučedníka',
+    marcelina_darowska_religious: 'Bl. Marceliny Darowska, řeholnice',  // TODO: Is `Darowska` (i.e. Polish surnames) inflected or not?
```
Polish names are usually inflected: Darowské

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    mary_assunta_pallotta_virgin: 'Bl. Marie Assunty Pallottové, panny',  // TODO: Is is `Pallotta` or `Pallottová`? In either case, is it inflected?
+    marydolores_rodriguez_sopena_virgin: 'Bl. Marie Dolores Rodríguez Sopeñové, panny', // TODO: Is is `Sopeña` or `Sopeñová`? In either case, is it inflected?
+    mary_stella_of_the_blessed_sacrament_mardosewicz_and_companions_virgins: 'Bl. Marie Stelly od Nejsvětější Svátosti Mardosewicz a druhů, panen a mučednic',
+    mary_theresa_ledochowska_virgin: 'Bl. Marie Terezie Ledóchowska, panny',  // TODO: Is `Ledóchowska` (i.e. Polish surnames) inflected or not?
```
Ledóchowské

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    mary_stella_of_the_blessed_sacrament_mardosewicz_and_companions_virgins: 'Bl. Marie Stelly od Nejsvětější Svátosti Mardosewicz a druhů, panen a mučednic',
+    mary_theresa_ledochowska_virgin: 'Bl. Marie Terezie Ledóchowska, panny',  // TODO: Is `Ledóchowska` (i.e. Polish surnames) inflected or not?
+    mary_theresa_chiramel_mankidiyan_virgin: 'Sv. Marie Terezie Chiramel Mankidiyan, panny',
+    marie_anne_blondin_virgin: 'Bl. Marie Anny Blondin, panny',  // TODO: Is it `Blondin` or `Blondiové`?
```
I think it's OK as is

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    marie_anne_blondin_virgin: 'Bl. Marie Anny Blondin, panny',  // TODO: Is it `Blondin` or `Blondiové`?
+    marie_leonie_paradis_virgin: 'Bl. Marie Léonie Paradis, panny',  // Note: I could not find her name online on Czech websites
+    marie_rose_durocher_virgin: 'Bl. Marie Rose Durocher, panny',
+    mary_angela_truszkowska_virgin: 'Bl. Marie Anděly Truszkowska, panny',  // TODO: Should the surname be inflected?
```
Truszkowské

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    marie_leonie_paradis_virgin: 'Bl. Marie Léonie Paradis, panny',  // Note: I could not find her name online on Czech websites
+    marie_rose_durocher_virgin: 'Bl. Marie Rose Durocher, panny',
+    mary_angela_truszkowska_virgin: 'Bl. Marie Anděly Truszkowska, panny',  // TODO: Should the surname be inflected?
+    mary_of_jesus_crucified_petkovic_virgin: 'Bl. Marie od Ukřižovaného Ježíše Petković, panny',
```
Croatian, but in this case (a woman with a name with what a Czech ear considers a "male" suffix) it's OK uninflected.

tukusejssirs replied:
Do you think that it would be better to add the `-ová` suffix?

igneus replied:
No. I tried to say that it's right *not* to add it in this case (unlike Croatian male names, for which I generally suggested inflecting).

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    marie_rose_durocher_virgin: 'Bl. Marie Rose Durocher, panny',
+    mary_angela_truszkowska_virgin: 'Bl. Marie Anděly Truszkowska, panny',  // TODO: Should the surname be inflected?
+    mary_of_jesus_crucified_petkovic_virgin: 'Bl. Marie od Ukřižovaného Ježíše Petković, panny',
+    mary_of_jesus_the_good_shepherd_siedliska_virgin: 'Bl. Marie od Ježíše Dobrého Pastýře Siedliska, panny',  // TODO: (1) `Dobrý Pastýř` or `dobrý Pastýř` (uppercase or lowercase `d`); (2) Should the surname be inflected?
```
probably Siedliské, but I'm not 100% sure

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    peter_kibe_priest_and_companions_martyrs: 'Bl. Petra Kibe, kněze, a druhů, mučedníků',
+    peter_to_rot_martyr: 'Bl. Petra To Rot, mučedníka',
+    pius_ix_pope: 'Bl. Pia IX., papeže',
+    raphael_chylinski_priest: 'Bl. Rafaela Chyliński, kněze',
```
Chylińského

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    andrew_apostle: 'Sv. Ondřeje, apoštola',
+    andrew_apostle_patron_of_russia: 'Sv. Ondřeje, apoštola a patrona Ruska',
+    angela_merici_virgin: 'Sv. Anděly Mericiové, panny',
+    joachim_and_anne_patroness_of_the_province_of_quebec_parents_of_mary: 'Sv. Anny, patronky provincie Québec, Jáchyma, rodičů Panny Marie',
```
Probably
Sv. Anny, patronky provincie Québec, **a** Jáchyma, rodičů Panny Marie

tukusejssirs replied:
Good catch! :+1:

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+  //   gabriel_taurin_dufresse_bishop: 'Saint Gabriel-Taurin Dufresse, Bishop and Martyr',
+  //   gall_of_switzerland_abbot: 'Saint Gall, Abbot and Missionary',
+  //   genevieve_of_paris_virgin: 'Saint Genevieve, Virgin',
+    george_of_lydda_martyr: 'Sv. Jiřího, mučedníka',
```
(definitely) sv. Jiří

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+  //   ita_of_killeedy_virgin: 'Saint Ita, Virgin',
+  //   ivo_of_kermartin_priest: 'Saint Ivo, Priest',
+    james_apostle: 'Sv. Jakuba, apoštola',
+    james_apostle_patron_of_spain: 'Sv. Jakuba, apoštola a patrona Španělska',
```
Usually there's no "a" before patronal titles:
`Sv. Jakuba, apoštola, patrona Španělska`
(I don't repeat this for every patronal title, but it's a general issue)

tukusejssirs replied:
I consider this a non-issue _per se_, though it is a _style issue_. A style that depends on many factors and I have seen with and without a conjunction. Maybe once, romcal will support both (or multiple) styles after personal metadata separation from the celebration names (see #154).

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+  //   marguerite_bourgeoys_virgin: 'Saint Marguerite Bourgeoys, Virgin',
+  //   marguerite_dyouville_religious: 'Saint Marguerite d’Youville, Religious',
+  //   mary_of_jesus_in_the_blessed_sacrament_venegas_virgin: 'Saint Mary of Jesus in the Blessed Sacrament Venegas, Virgin',
+    maria_goretti_virgin: 'Saint Maria Goretti, Virgin and Martyr',
```
uncommented, but untranslated

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    teresa_benedicta_of_the_cross_stein_virgin: 'Sv. Terezie Benedikty od Kříže, panny a mučednice',
+    teresa_of_calcutta_religious: 'Sv. Terezy z Kalkaty, řeholnice',
+  //   teresa_of_jesus_jornet_ibars_virgin: 'Saint Teresa of Jesus Jornet Ibars, Virgin',
+    teresa_of_jesus_of_avila_virgin: 'Sv. Terezie od Ježíše z Ávily, panny a učitelky církve',
```
I understand the combined title "od Ježíše z Ávily" is your editorial decision (I don't remember where exactly, but it was discussed somewhere quite recently), but *noone ever* calls her like this in Czech Republic. It looks and sounds very strange. Either "Sv. Terezie od Ježíše", or "Sv. Terezie z Ávily", never both.

tukusejssirs replied:
True, it is my editorial decision, because we (in romcal) try to support all locales and all calendars, and as a matter of fact, there are multiple (at least three) St Teresias with the name `St Teresia of [Child] Jesus`. IMO, we need to be able to exactly know of which Teresia is the celebration named.

> but _noone ever_ calls her like this in Czech Republic.

Same here, in Slovakia and I think anywhere else, _where no other St Teresia of Jesus is celebrated_.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+    our_lady_of_sorrows_patroness_of_slovakia: 'Panny Marie Bolestné, patronky Slovenska',
+  //   george_of_lydda_martyr_patron_of_england: 'Saint George, Martyr and Patron of England',
+    benedict_of_nursia_abbot_patron_of_europe: 'Sv. Benedikta, opata, patrona Evropy',
+    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_copatrons_of_europe: 'Sv. Cyrila, mnicha, a Metoděje, biskupa, spolupatronů Evropy',
```
patrons of Europe are in Czech always all called "patrons", never "copatrons" (spolupatroni).

Moreover, you don't use the title everywhere the same way, evoking a (false) impression that there is a hierarchy among "patrons of Europe" ("patrons" superior to "copatrons").

tukusejssirs replied:
> patrons of Europe are in Czech always all called "patrons", never "copatrons" (spolupatroni).

It might be so, however, they are _co-patrons_ ([src](http://www.vatican.va/content/john-paul-ii/en/motu_proprio/documents/hf_jp-ii_motu-proprio_01101999_co-patronesses-europe.html)).

> Moreover, you don't use the title everywhere the same way

What do you mean by this? If I made a mistake somewhere, just tell me, I’ll fix it. Only St Benedict of Nursia is the (principal) patron of Europe, Sts Cyril and Methodius, St Bridget of Sweden, St Catherine of Siena and St Edith Stein are co-patron(esse)s.

igneus replied:
> Only St Benedict of Nursia is the (principal) patron of Europe, Sts Cyril and Methodius, St Bridget of Sweden, St Catherine of Siena and St Edith Stein are co-patron(esse)s.

That's surprising for me. Eventually I'll try to read a bit more on this topic and either correct my misconception or refute yours. For the time being please consider my comment starting this thread void.

tukusejssirs replied:
I am not a theologian nor liturgist, just a mere enthusiast, who uses the Internet as much as possible to get the information I need. :smiley: Therefore, don’t take the information I provide you as 100 % correct. :smiley:

> Eventually I'll try to read a bit more on this topic and either correct my misconception or refute yours.

Please do so. I might learn something more from you, however, I think that those patronage divisions (or hierarchy) are correct. There might be even more than one (principal) patronages for a particular territory (e.g. a country).

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,798 @@
+  //   francis_of_assisi_patron_of_italy: 'Saint Francis of Assisi, Patron of Italy',
+    adalbert_of_prague_bishop_patron_of_poland: 'Sv. Vojtěcha, biskupa, mučedníka a patrona Polska',
+  //   stanislaus_of_szczepanow_bishop_patron_of_poland: 'Saint Stanislaus, Bishop, Martyr and Patron of Poland',
+  wenceslaus_i_of_bohemia_martyr_patron_of_czech_republic: 'Sv. Václava, mučedníka, hlavního patrona českého národa',  // TODO: Add this to all other locales + czechRepublic calendar
```
`wenceslaus_i_of_bohemia_martyr_patron_of_czech_republic` - the identifier is inaccurate. St. Wenceslas is not really patron of the *Czech Republic* (the Czech state is non-confessional and has never elected a patron saint), he is a patron of the Czech *nation*. And the Czech word "národ", probably more than the English "nation", refers to an entity primarily ethnical rather than political.
(Which is also the reason why I am not excited that we have an official patron of the Czech nation, as mentioned in igneus/calendarium-romanum#72 , and prefer the older patronal titles referring to a region/territory rather than an ethnical group.)

tukusejssirs replied:
I understand the Czech _words_ and their (primary) meaning, however, those patronage titles you have are quite complicated from my point of view. :wink:

In romcal, we currently support only _nation-wide_/_country-wide_ calendars, no diocesian one. Those would be quite nice addition to romcal, but they would require to be implemented (see #48).

Therefore, from all those patronage titles you have in the Czech proper, which are _for the whole country_ (i.e. Czech Republic)? Are there none? From igneus/calendarium-romanum#72 I understood that St Wenceslaus a patron of all Czech Republic (as in political country), but now you say this isn’t quite true.

Is he just a patron of the the Bohemia (_Česko_, excluding Moravia and Sliezko), or maybe the Bohemian province?

igneus replied:
The issue about St. Wenceslas is not that his patronage would not be country-wide, but that it's object is "český _národ_" (Czech nation): not a political entity (Czech Republic), but an ethnical one.

tukusejssirs replied:
Argh, now I see what you meant.

Now that complicates some stuff in romcal (e.g. #201), but you are right.

So, St Wenceslaus is patron fo the Czech nation; does it include the Moravians too? Because from my point of view, Czechs are a different nation from Moravians; am I wrong?

igneus replied:
As discussed in #72, for this purpose Moravians are definitely considered part of the Czech nation. There may be some people asserting a Moravian nation separate from the Czech one, but this way of thinking (is not mainstream and) was definitely not involved when the calendar of Czech and Moravian dioceses was being conceived.

tukusejssirs replied:
Okay, thanks for the thorough explanation.

I’ll use `patron_of_czech_nation` key name suffix.

Anyway, I am tempted to omit `hlavního` from the key value, because I think that in Czech (like in Slovak), the `(principal) patron` is localised simply as `patron` and `co-patron` as `spolupatron`. However, I see a counter-argument that the Czech liturgical books use `hlavní patron` for St Wenceslaus; is `co-patron` localised simply as `patron`? Could check this in the Czech liturgical books please?

tukusejssirs replied:
Just a side question: I have just noticed in the [JPII motu proprio (Latin)](http://www.vatican.va/content/john-paul-ii/la/motu_proprio/documents/hf_jp-ii_motu-proprio_01101999_co-patronesses-europe.html), they used the term _compatronus_, while I thought that _copatronus_ is correct. In the [Latin locale](https://github.com/romcal/romcal/blob/_F-TypeScript/src/locales/la.ts#L473), I used the term _patronus secundarius_. Which is correct according to you

igneus replied:
> Just a side question: I have just noticed in the JPII motu proprio (Latin), they used the term compatronus, while I thought that copatronus is correct. In the Latin locale, I used the term patronus secundarius. Which is correct according to you

No idea, sorry. It's quite possible that both forms are correct.

> Anyway, I am tempted to omit hlavního from the key value, because I think that in Czech (like in Slovak), the (principal) patron is localised simply as patron and co-patron as spolupatron. However, I see a counter-argument that the Czech liturgical books use hlavní patron for St Wenceslaus; is co-patron localised simply as patron? Could check this in the Czech liturgical books please?

The calendarium-romanum data file you worked with literally reproduces the calendar as printed at the beginning of the Czech breviary and the patronage titles are the same in all currently used liturgical books.

To my knowledge, calendar and liturgical books of Czech and Moravian dioceses don't know an explicit distinction between "patron" and "copatron/secondary patron". The proper calendar calls several saints principal patrons (`hlavní patron` in Czech, `patronus principalis` in [Latin](https://kpmk.eu/kpmkeu/index.php/liturgie/liturgicke-texty/147-texty-narodniho-propria-latinsky)) of various subjects (nation, the three historical countries, dioceses) and there is no second class of patrons in use.

Titles of (all) the feasts of patrons of Europe (which are not proper to the Czech calendar, but shared by all European particular calendars) are, on the other hand, translated simply `patron Evropy`. My understanding has always been that "an equal level of patronage" is shared by all of them and therefore there are only "patroni Evropy" and no "hlavní patron Evropy", but as discussed earlier today, I have yet to clear this question for myself. Currently I cannot exclude the possibility that St. Benedict is actually principal patron of Europe (as you stated earlier today) and official wording of the Czech liturgical calendar for some reason just neglects this distinction.

Either of `patron/spolupatron` or `hlavní patron/patron` is viable, I would probably prefer the second option, as it is a bit closer to the wording used in official liturgical books and clearly distinguishes the principal patron.

tukusejssirs replied:
Thanks for the answer.

> Either of `patron/spolupatron` or `hlavní patron/patron` is viable, I would probably prefer the second option, as it is a bit closer to the wording used in official liturgical books and clearly distinguishes the principal patron.

I see your reasoning. Just a note: in Slovak liturgical books, we use the former, i.e. `patrón` and `spolupatrón`. We clearly distinguish between patrons of Europe that way.

igneus replied:
(Off topic: they/you've had [printed post-conciliar Latin proper since 1977](https://christ-net.sk/node/3671)! Much envy here.)

igneus reviewed (commented):
Reviewed.

There are several TODO notes which I arbitrarily ignored, since I don't know anything about the feasts/saints in question and/or best Czech rendering of their names, and don't really care. (If anyone ever happens to generate some exotic foreign calendar in Czech and is scandalized by incorrect name of his favourite saint, he can open a PR with a fix himself. But I don't expect this to ever happen.)

igneus commented:
I should add that my review here also has a rather low authority / guarantee of correctness / ...
I reviewed in a fast manner, checking for stuff obviously incorrect or strange, using only my "best knowledge" of my mother tongue. But my "best knowledge" is rather poor. I have never drilled the more obscure Czech language skills like aspiring Czech philologists or journalists do and even the rules of thumb I pronounced in some comments (like "Polish/Croatian surnames are usually inflected in Czech") are based only on experience with the language.

tukusejssirs commented:
> I should add that my review here also has a rather low authority / guarantee of correctness / ...
> I reviewed in a fast manner, checking for stuff obviously incorrect or strange, using only my "best knowledge" of my mother tongue. But my "best knowledge" is rather poor. I have never drilled the more obscure Czech language skills like aspiring Czech philologists or journalists do and even the rules of thumb I pronounced in some comments (like "Polish/Croatian surnames are usually inflected in Czech") are based only on experience with the language.

Thanks, @igneus, IMO it is perfect enough for our current needs. :+1:

tukusejssirs commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+  //   shipwreck_of_saint_paul_apostle: 'Shipwreck of Saint Paul, Apostle',
+  //   dunstan_of_canterbury_bishop: 'Saint Dunstan, Bishop',
+  //   sunday_of_the_word_of_god: 'Sunday of the Word of God',
+    passion_of_saint_john_the_baptist: 'Umučení sv. Jana Křtitele',
```
Most of the celebrations of some events (i.e. non-personal celebrations like the _Passion of St John the Baptist_) or religious names (like _XXX of St YYY_) in the Czech locale output `svatý` in full; then why is this abbreviated?

You told me today that you used the official wording from the Czech Liturgy of the Hours, however, I see this as rather inconsistent. IMHO either always write it out in full or always abbreviate it. In general: be consistent and create some rules (though that might be an issue for the Liturgical Commission of the Czech episcopal conference, not you personally).

igneus replied:
For `calendarium-romanum` I reproduce the calendar as printed in liturgical books (i.e. I literally reproduce an official "document") and I don't intend to make any editorial improvements.
Most probably there are no (important) reasons why an abbreviation was used here. Maybe it's a random inconsistency, maybe it was motivated by some typographical concerns. I don't care.

I understand that `romcal` attitude to matters like this is different - please *"be consistent and create some rules"* as you wish.

tukusejssirs replied:
I see you are less pedantic in such things (as most people are). :smiley: _Carelessness_ (as in _I don’t care_) drives me crazy, however, I know that this inconsistency is not the most important thing to deal with, but for me it is one of many issues.

Nevertheless, I think the best would be either to abbreviated all canonisation levels or none. Later on, when #241 and #238 are implemented, one could choose either full form or the abbreviated one.

Anyway, thanks for the feedback! Hopefully, I’ll finish off this PR later this week.

tukusejssirs commented:
@igneus, in the latest commit, I have fixed all _resolved_ issues found by you, removed resolved TODOs and localised all celebrations of Our Lady and some more (approx L207-L340, which include some strings that were already localised). Although I have removed some TODOs, I have kept back those, which I am still not quite sure if there are correct. Please do comment on each when you’ll find some time. Thanks. :smiley:

igneus commented on src/locales/cs.ts:
```diff
@@ -93,81 +93,81 @@ export default {
     albertina_berkenbrock_virgin: 'Bl. Albertiny Berkenbrockové, panny a mučednice',
     aloysius_stepinac_bishop: 'Bl. Aloise Stepinace, biskupa a mučedníka',
-    andre_grasset_priest: 'Bl. Ondřeje Grasset, kněze a mučedníka',  // TODO: Check the surname inflection
+    andre_grasset_priest: 'Bl. Ondřeje Grasset, kněze a mučedníka',
```
Seeing this now, the inflected form `Grasseta` would probably be used

igneus commented on src/locales/cs.ts:
```diff
@@ -180,18 +180,18 @@ export default {
     charles_spinola_and_jerome_de_angelis_priests: 'Bl. Karla Spinoly a Jeronýma de Angelis, kněží a mučedníků',
     sebastian_de_aparicio_religious: 'Bl. Šebestiána de Aparicio, řeholníka',
-    nykyta_budka_and_vasyl_velychkovsky_bishops: 'Bl. Nikity Budka a Vasila Velichkovského, biskupů a mučedníků',  // TODO: Do proof their full names
+    nykyta_budka_and_vasyl_velychkovsky_bishops: 'Bl. Nikity Budku a Vasila Velichkovského, biskupů a mučedníků',  // TODO: Do proof their full names
```
If `Budka` is nominative, Czech genitive is definitely not `Budku`.

Either `Budky` or uninflected `Budka`, which would, however, make the reader wrongly assume that nominative is `Budek`.

igneus commented on src/locales/cs.ts:
```diff
@@ -204,141 +204,140 @@ export default {
+    finding_of_the_holy_cross: 'Nalezení svatého Kříže',
     first_martyrs_of_the_holy_roman_church: 'Svatých prvomučedníků římských',
-  //   guardian_angel_of_portugal: 'Guardian Angel of Portugal',
+    guardian_angel_of_portugal: 'Anděla strážného Portugalska',  // TODO: Proof the inflection
```
inflection is OK

igneus commented on src/locales/cs.ts:
```diff
@@ -204,141 +204,140 @@ export default {
-  //   our_lady_of_knock: 'Our Lady of Knock',
-  //   our_lady_of_lanka: 'Our Lady of Lanka',
-  //   our_lady_of_lebanon: 'Our Lady of Lebanon',
+    our_lady_of_hungary_patroness_of_hungary: 'Panny Marie Maďarů, patronky Maďarů',
```
This may be correct, but sounds very strange.

Also the patronage titles in English and Czech do not match exactly: is she patroness of the people (Hungarians, Maďaři), or the country (Hungary, Maďarsko)?

tukusejssirs replied:
Well, the original (Hungarian) name of the celebration is [`Szűz Mária, Magyarok Nagyasszonya, Magyarország Főpátrónája`](http://tar.liturgia.hu/Naptar/dir2019_nyomdai_withoutborder.pdf#page=183), which _literally_ means `Virgin Mary, Our Lady of Hungarians, Principal Patron of Hungary`, but I would translate it into English simply as `Our Lady of Hungary, Patron of Hungary`.

When I search the Internet for the Czech translation, I found somewhere `patronky Maďarů`, however, you are right that it is not correct. I’ll change it to `patronky Maďarska`.

On the other hand, I must say that I personally (but probably Slovaks in general) does not really distinguish between patrons of a country’s majority nation and patrons of that country; that is patron of Slovakia for me/us equals patron of the Slovak nation. Although I see the difference when I look at it from a literal, linguistic point of view (as you and the people in the Czech Republic view it).

igneus commented on src/locales/cs.ts:
```diff
@@ -204,141 +204,140 @@ export default {
+    our_lady_queen_of_poland: 'Panny Marie, Královny Polska',
     our_lord_jesus_christ_the_eternal_high_priest: 'Ježíše Krista, nejvyššího a věčného kněze',
-  //   our_lord_of_miracles: 'Our Lord of Miracles',
+    our_lord_of_miracles: 'Pána zázraků',
```
Sounds strange.

tukusejssirs replied:
Indeed it sounds. Just look for it on the Internet. Original (Spanish) celebration name is `Señor de los Milagros`, i.e. `Lord of the Miracles`.

On the Internet, I have also found it localised into Czech as `Mistr zázraků` (there was a film created with that name too), but that sounds even strager to me.

igneus replied:
I can't comment the Spanish, since I don't speak the language and haven't read much in it, but there is a clear difference between the English `Our Lord of Miracles` and Czech `Pán(a) Zázraků`: "*Our* Lord" (at least written like this) is a devotional title used in Catholic contexts exclusively for Jesus Christ, c.f. similar "Our Lady". Czech "Pán" is far less exclusive and the construct "Pán" + genitive, relating to Jesus Christ, has no tradition in the Czech Catholic language use. Parallels it evokes are completely secular - I first think of "Pán prstenů" (and "Pán pařmenů").

But I probably don't have a better suggestion. Maybe I would add the original Spanish title in parentheses, like `Pána zázraků (Señor de los Milagros)`.

tukusejssirs replied:
> Czech "Pán" is far less exclusive and the construct "Pán" + genitive, relating to Jesus Christ, has no tradition in the Czech Catholic language use. Parallels it evokes are completely secular - I first think of "Pán prstenů" (and "Pán pařmenů").

I agree with you that `Pán zázraků` has no tradition in Czech (or Slovak) liturgical usage, however, I still think that it sounds better than `Náš Pán zázrakov` or anything else.

> But I probably don't have a better suggestion. Maybe I would add the original Spanish title in parentheses, like `Pána zázraků (Señor de los Milagros)`.

Thanks for the suggestion, however, personally I don’t like appending anything the a title.

---

For now, I keep it as is.

igneus commented on src/locales/cs.ts:
```diff
@@ -204,141 +204,140 @@ export default {
-  //   aelred_of_rievaulx_abbot: 'Saint Aelred of Rievaulx, Abbot',
-  //   aengus_of_tallaght_bishop: 'Saint Aengus, Bishop and Abbot',
+    aelred_of_rievaulx_abbot: 'Sv. Aelreda z Rievaulx, opata',
+    aengus_of_tallaght_bishop: 'Sv. Aengusa, biskupa a opata',
```
If nominative is `Aengus` (Latin or at least latinized name of 2nd declension), classical Czech genitive would be `Aenga`, younger people may prefer `Aenguse`.

tukusejssirs replied:
Which one do you personally prefer? Or another way: which one do you think would better fit into the current Czech localisation of the celebration names used in the Czech Proper?

igneus replied:
My bet would be the more classical form: `Sv. Aenga, biskupa a opata`

igneus commented on src/locales/cs.ts:
```diff
@@ -204,141 +204,140 @@ export default {
     ambrose_of_milan_bishop: 'Sv. Ambrože, biskupa a učitele církve',
-  //   andre_bessette_religious: 'Saint André Bessette, Religious',
-  //   andrew_bobola_priest: 'Saint Andrew Bobola, Priest and Martyr',
+    andre_bessette_religious: 'Svatého Andreje Alfréda Bessette, řeholníka',
```
I'm not sure if the French name `André` would be bohemized, but if so, the correct Czech version is `Ondřeje`

tukusejssirs replied:
I think yes, it should be bohemised (I did it by mistake, because in Slovak, we you both `Andrej` and `Ondrej` form).

Anyway, should the surname be inflected, just like [`Grasset`](https://github.com/romcal/romcal/pull/248#discussion_r511580004)’s name?

igneus commented on src/locales/cs.ts:
```diff
@@ -204,141 +204,140 @@ export default {
-  //   benno_of_meissen_bishop: 'Saint Benno of Meissen, Bishop',
-  //   bernadette_soubirous_virgin: 'Saint Bernadette Soubirous, Virgin',
+    benno_of_meissen_bishop: 'Sv. Benna z Míšně, biskupa',
+    bernadette_soubirous_virgin: 'Sv. Bernadety Soubirousové, panny',  // TODO: Or rather `Soubirous`?
```
Both are in use, but `Sv. Bernadety Soubirous` (uninflected) seems to be more wide-spread and I would prefer it.

igneus commented on src/locales/cs.ts:
```diff
@@ -204,141 +204,140 @@ export default {
+    bruno_of_querfurt_bishop: 'Sv. Bruna z Querfurtu, biskupa a mučedníka',
     bruno_of_cologne_priest: 'Sv. Bruna, kněze',
-  //   caesarius_of_arles_bishop: 'Saint Caesarius of Arles, Bishop',
+    caesarius_of_arles_bishop: 'Sv. Césara z Arles, biskupa',
```
`Césaria`

igneus commented on src/locales/cs.ts:
```diff
@@ -204,141 +204,140 @@ export default {
     christopher_magallanes_priest_and_companions_martyrs: 'Sv. Kryštofa Magallanese, kněze, a druhů, mučedníků',
-  //   christopher_of_palestine_martyr: 'Saint Christopher, Martyr',
-  //   ciaran_of_clonmacnoise_abbot: 'Saint Ciaran, Abbot',
+    christopher_of_palestine_martyr: 'Sv. Krištofa, mučedníka',
```
Kryštofa

igneus reviewed (commented):
TODOs I didn't comment on are from the "sorry, no idea" category.

tukusejssirs commented:
@igneus, I have fixed some of the the issues you have raised (those I’ve marked as resolved) and localised some more strings (L344-388).

tukusejssirs commented:
@igneus, I have just pushed a new commit, which contains localised strings up to L555. All in all, 560 sanctorale keys are localised of 700 sanctorale keys, which makes 80 %.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    cyril_of_jerusalem_bishop: 'Sv. Cyrila Jeruzalémského, biskupa a učitele církve',
+    damasus_i_pope: 'Sv. Damasa I. papeže',
+    damien_de_veuster_priest: 'Sv. Damiána de Veuster, kněze',
+    david_of_wales_bishop: 'Sv. Davida Velšského, biskupa',  // TODO: `Velšský` or `z Walesu`?
```
Oboje je víceméně stejně dobré, i když "Velšský" dnes možná zní už trochu archaicky.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    damasus_i_pope: 'Sv. Damasa I. papeže',
+    damien_de_veuster_priest: 'Sv. Damiána de Veuster, kněze',
+    david_of_wales_bishop: 'Sv. Davida Velšského, biskupa',  // TODO: `Velšský` or `z Walesu`?
+    david_lewis_priest: 'Sv. Davida Lewise, kněze a mučedníka',  // TODO: `Lewis` or `Lewise`?
```
"Lewise" je v pořádku

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    damien_de_veuster_priest: 'Sv. Damiána de Veuster, kněze',
+    david_of_wales_bishop: 'Sv. Davida Velšského, biskupa',  // TODO: `Velšský` or `z Walesu`?
+    david_lewis_priest: 'Sv. Davida Lewise, kněze a mučedníka',  // TODO: `Lewis` or `Lewise`?
+    davnet_of_sliabh_beagh_virgin: 'Sv. Davnety, panny',  // TODO: Check the inflection
```
Vypadá ok

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    deiniol_of_bangor_bishop: 'Sv. Deiniola, biskupa',
+    demetrius_of_thessaloniki_martyr: 'Sv. Demetria Soluňského, mučedníka',  // TODO: `Dimitria` or `Demetra` or `Demetria`?
+    denis_of_paris_bishop_and_companions_martyrs: 'Sv. Dionýsia, biskupa, a druhů, mučedníků',
+    dionysius_the_areopagite_bishop: 'Sv. Dionysia Areopagity, biskupa a mučedníka',
```
délky nekonsistentní s předchozím řádkem

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    denis_of_paris_bishop_and_companions_martyrs: 'Sv. Dionýsia, biskupa, a druhů, mučedníků',
+    dionysius_the_areopagite_bishop: 'Sv. Dionysia Areopagity, biskupa a mučedníka',
+    dominic_de_guzman_priest: 'Sv. Dominika, kněze',
+    dyfrig_of_wales_bishop: 'Sv. Dubricia Velšského, biskupa',  // TODO: `Velšský` or `z Walesu`?
```
viz výše

tukusejssirs replied:
Just a question: do you think it is `Dubricius` or `Dubrícius`? (Note the acute accent on `i`.)

tukusejssirs replied:
Ping @igneus.

igneus replied:
Definitely short `Dubricius`. (`Dubrícius` looks very Slovak. In Czech the first `i` would often also be pronounced long, but not written as such.)

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    dominic_de_guzman_priest: 'Sv. Dominika, kněze',
+    dyfrig_of_wales_bishop: 'Sv. Dubricia Velšského, biskupa',  // TODO: `Velšský` or `z Walesu`?
+    edmund_campion_priest: 'Sv. Edmunda Kampiána, kněze a mučedníka',
+    edmund_of_abingdon_bishop: 'Sv. Edmunde z Abingdonu, biskupa',
```
Edmunda

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    dyfrig_of_wales_bishop: 'Sv. Dubricia Velšského, biskupa',  // TODO: `Velšský` or `z Walesu`?
+    edmund_campion_priest: 'Sv. Edmunda Kampiána, kněze a mučedníka',
+    edmund_of_abingdon_bishop: 'Sv. Edmunde z Abingdonu, biskupa',
+    edward_the_confessor: 'Sv. Edvarda Vyznavače',
```
určitě `Eduarda` ("Edvard" se používá jen u jmen, která se tak píší dlouho/odjakživa - vlastně neznám jiného "Edvarda", než čsl. prezidenta Edvarda Beneše)

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    edmund_of_abingdon_bishop: 'Sv. Edmunde z Abingdonu, biskupa',
+    edward_the_confessor: 'Sv. Edvarda Vyznavače',
+    elijah_prophet: 'Sv. Eliáše, proroka',
+    elizabeth_ann_seton_religious: 'Sv. Alžběty Anny Setonové, řeholnice',  // TODO: Should we bohemise her name?
```
Sorry for the Czech comments above. I forgot myself.

Yes, we should bohemise.

tukusejssirs replied:
> Sorry for the Czech comments above. I forgot myself.

At first, I was puzzled, but I had no problem with understanding your comments. :laughing:

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    emeric_of_hungary: 'Sv. Emericha',
+    enda_of_aran_abbot: 'Sv. Endy, opata',
+    ephrem_the_syrian_deacon: 'Sv. Efréma Syrského, jáhna a učitele církve',
+    eric_ix_of_sweden_martyr: 'Sv. Erika IX. Švédského, Martyr',
```
title untranslated

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    enda_of_aran_abbot: 'Sv. Endy, opata',
+    ephrem_the_syrian_deacon: 'Sv. Efréma Syrského, jáhna a učitele církve',
+    eric_ix_of_sweden_martyr: 'Sv. Erika IX. Švédského, Martyr',
+    etheldreda_of_ely_abbess: 'Sv. Eteldredy z Ely, opatky',
```
"abatyše" (not "opatky")

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    faustina_kowalska_virgin: 'Sv. Faustiny Kowalské, panny',
+    philip_of_jesus_de_las_casas_martyr: 'Sv. Filipa od Ježiša de las Casas, mučedníka',
+    ferdinand_iii_of_castile: 'Sv. Ferdinanda III. Kastilského',
+    fergal_of_salzburg_bishop: 'Sv. Virgila ze Salzburgu, biskupa a misionáře',  // TODO: `ze Salzburgu` or `solnohradského`? (src: http://librinostri.catholica.cz/download/VondruskaSvati2-OCR.pdf#page=217 … it might be an archaism though)
```
"Solnohradského" would be a harsh archaism

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    florian_of_lorch_and_companions_martyrs: 'Sv. Floriána a druhů, mučedníků',
+    florian_of_lorch_martyr: 'Sv. Floriána, mučedníka',
+    frances_of_rome_religious: 'Sv. Františky Římské, řeholnice',
+    frances_xavier_cabrini_virgin: 'Sv. Františky Xaviery Cabriniové, panny',  // TODO: `Cabriniové` or `Cabrini`? `Xaviera` or `Xavier`?
```
Definitely not "Xaviery" - I would go for "Xavery", although "Xaver" may be an option.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    florian_of_lorch_martyr: 'Sv. Floriána, mučedníka',
+    frances_of_rome_religious: 'Sv. Františky Římské, řeholnice',
+    frances_xavier_cabrini_virgin: 'Sv. Františky Xaviery Cabriniové, panny',  // TODO: `Cabriniové` or `Cabrini`? `Xaviera` or `Xavier`?
+    francis_borgia_priest: 'Sv. Františka Borgii, Priest',
```
Title untranslated.

A somewhat archaic, but still frequently used version, which I would prefer here, is `Sv. Františka Borgiáše`

tukusejssirs replied:
I’ve never heard `Borgiáš` word. :smiley: I somewhat don’t like it (because I don’t like localising/changing personal names and surnames in general), but I think you know better what is more often used or sounds better in Czech, therefore I accept your suggestion.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    francis_solanus_priest: 'Sv. Františka Solana, kněze',
+    francis_xavier_priest: 'Sv. Františka Xaverského, kněze',
+    francois_de_montmorency_laval_bishop: 'Sv. Františka de Montmorency-Laval, biskupa',
+    fridolin_of_sackingen_monk: 'Sv. Fridolina ze Säckingen, mnicha',
```
Fridolína

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    fursa_of_peronne_abbot: 'Sv. Fursy, opata a misionáře',
+    gabriel_taurin_dufresse_bishop: 'Sv. Gabriela Taurin Dufress, biskupa a mučedníka',
+    gall_of_switzerland_abbot: 'Sv. Havla, opata a misionáře',
+    genevieve_of_paris_virgin: 'Sv. Genovévy, panny',
```
Jenovéfy

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    george_preca_priest: 'Sv. Jiřího Preca, kněze',
+    gerard_of_csanad_bishop: 'Sv. Gerarda z Čanádu, biskupa a mučedníka',  // Note: He is also known as `Gerard [di] Sagredo`, but he was a bishop of Csanad, Hungary
+    germanus_of_auxerre_bishop: 'Sv. Germana z Auxerre, biskupa',
+    gertrude_of_nivelles_abbess: 'Sv. Gertrúdy z Nivelles, opatky',
```
Gertrudy

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    germanus_of_auxerre_bishop: 'Sv. Germana z Auxerre, biskupa',
+    gertrude_of_nivelles_abbess: 'Sv. Gertrúdy z Nivelles, opatky',
+    gertrude_the_great_virgin: 'Sv. Gertrudy, panny',
+    gobnait_of_ballyvourney_virgin: 'Sv. Gobnaity, panny',  // TODO: I think the nominative is Czech (and Slovak) is `Gobnaita`, however it might be left as `Gobnait`
```
OK as is

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    gertrude_of_nivelles_abbess: 'Sv. Gertrúdy z Nivelles, opatky',
+    gertrude_the_great_virgin: 'Sv. Gertrudy, panny',
+    gobnait_of_ballyvourney_virgin: 'Sv. Gobnaity, panny',  // TODO: I think the nominative is Czech (and Slovak) is `Gobnaita`, however it might be left as `Gobnait`
+    gundisalvus_garcia_martyr: 'Sv. Gundisalva García, mučedníka',  // Note: He is one of the companions of St Paul Miki
```
I'm not sure if "García" should be inflected "Garcíi" or not

tukusejssirs replied:
While searching the Internet, I have found both forms. Personally, I’m not able to decide which one is better.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    hermann_joseph_of_steinfeld_priest: 'Sv. Heřmana Josefa, kněze',
+    hermenegild_the_visigoths_martyr: 'Sv. Hermenegilda, mučedníka',
+    hilary_of_poitiers_bishop: 'Sv. Hilaria, biskupa a učitele církve',
+    hilda_of_whitby_abbess: 'Sv. Hildy, opatky',
```
abatyše

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    hermenegild_the_visigoths_martyr: 'Sv. Hermenegilda, mučedníka',
+    hilary_of_poitiers_bishop: 'Sv. Hilaria, biskupa a učitele církve',
+    hilda_of_whitby_abbess: 'Sv. Hildy, opatky',
+    hildegard_of_bingen_abbess: 'Sv. Hildegardy z Bingen, opatky a učitelky církve',
```
abatyše

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    hyacinth_of_poland_priest: 'Sv. Hyacinta, kněze',
+    ignatius_of_antioch_bishop: 'Sv. Ignáce Antiochijského, biskupa a mučedníka',
+    ignatius_of_loyola_priest: 'Sv. Ignáce z Loyoly, kněze',
+    ildephonsus_of_toledo_bishop: 'Sv. Ildefonza z Toleda, biskupa',
```
Ildefonse

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    irenaeus_of_lyon_bishop: 'Sv. Ireneje, biskupa a mučedníka',
+    irene_of_macedonia: 'Sv. Ireny',
+    isidore_of_seville_bishop: 'Sv. Izidora, biskupa a učitele církve',
+    isidore_the_farmer: 'Sv. Izidora z Madridu, rolníka',  // TODO: Or `Izidora z Madridu` or `Izidora Madriského` or `Izidora Rolníka`? `Isidor` or `Izidor`? (src: https://cs.wikipedia.org/wiki/Isidor_z_Madridu)
```
Both "Isidor" and "Izidor" are used - OK as is

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    john_jones_priest: 'Sv. Jana Jonese, kněze a mučedníka',  // src: http://www.sfr.cz/breviar_detail.aspx?a=317
+    john_leonardi_priest: 'Sv. Jana Leonardiho, kněze',
+    john_macias_religious: 'Sv. Jana Macíase, řeholníka',  // src: http://breviar.op.cz/?a=7&datum=2017-09-18&k=18SEP&chyba=1
+    john_mary_vianney_priest: 'Sv. Jana Marie Vianneye, kněze',
```
Jana *Maria* Vianneye

I haven't ever seen or heard the name with "Marie"

tukusejssirs replied:
This one comes from your own codebase. :smiley: → [`czech-cs.txt` (L148)](https://github.com/igneus/calendarium-romanum/blob/master/data/czech-cs.txt#L148)

tukusejssirs replied:
Ping, @igneus. Do we both have it incorrectly or is it actually correct?

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    josaphat_kuntsevych_bishop: 'Sv. Josafata, biskupa a mučedníka',
+    joseph_de_anchieta_priest: 'Sv. Josefa de Anchieta, kněze',
+    joseph_bilczewski_bishop: 'Sv. Josefa Bilczewského, biskupa',
+    jose_maria_de_yermo_y_parres_priest: 'Sv. Josefa Marie de Yermo y Parres, kněze',
```
Here I would probably leave the whole name in it's Spanish form

tukusejssirs replied:
Searching `site:.cz de Yermo knez` on the Internet, I can only find `Josef de Yermo`, occasionally `Josef Maria de Yermo`. That said, I don’t consider those results as the primary sources nor the correct ones.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    joseph_de_anchieta_priest: 'Sv. Josefa de Anchieta, kněze',
+    joseph_bilczewski_bishop: 'Sv. Josefa Bilczewského, biskupa',
+    jose_maria_de_yermo_y_parres_priest: 'Sv. Josefa Marie de Yermo y Parres, kněze',
+    josemaria_escriva_de_balaguer_priest: 'Sv. Josemaríe Escrivá de Balaguer, kněze',
```
I'm not too acquainted with this corner of the church, but I believe the first name is usually left uninflected and the second one is being inflected:
`Sv. Josemaría Escrivy de Balaguer, kněze`
cf. https://zpravy.proglas.cz/udalosti/opus-dei-ma-noveho-prelata-fernardo-ocariz-bra-o/
https://paloucek.wixsite.com/paloucek/about

tukusejssirs replied:
Your first link (zpravy.proglas.cz) contains that name in genitive as `svatého Josemaria Escrivy`, however, the other link (paloucek.wixsite.com) contains `Josemaríi Escrivu`:
- papež Jan Pavel II. monsignora _**Escrivu**_ na Svatopetrském náměstí v Římě svatořečil;
- Katolická církev slaví svátek sv. _**Josemaríi**_;
- Prvním nástupcem sv. _**Josemaríi**_ v čele Opus Dei;
-  stal se nejbližším spolupracovníkem svatého _**Josemaríi**_.

(Correct me if I’m wrong.)

tukusejssirs replied:
Ping, @igneus.

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    joseph_zhang_dapeng_martyr: 'Sv. Josefa Zhang Dapeng, mučedníka',
+    joseph_sebastian_pelczar_bishop: 'Sv. Josefa Šebestiána Pelczara, biskupa',  // `Pelczara`: src: http://librinostri.catholica.cz/download/PelczJKVzAMiK1az3-r0.pdf#page=5
+    juan_diego_cuauhtlatoatzin: 'Sv. Jana Didaka Cuauhtlatoatzina',
+    juliana_of_liege_virgin: 'Sv. Juliány z Liège, panny',  // TODO: `Juliana` or `Juliána`? … `Liège` or `Lutych`? (src: https://cs.wikipedia.org/wiki/Lutych) Note that in Slovak, `Lutych` is considered an archaic name for the city, while `Liège` is considered correct.
```
OK as is

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    joseph_sebastian_pelczar_bishop: 'Sv. Josefa Šebestiána Pelczara, biskupa',  // `Pelczara`: src: http://librinostri.catholica.cz/download/PelczJKVzAMiK1az3-r0.pdf#page=5
+    juan_diego_cuauhtlatoatzin: 'Sv. Jana Didaka Cuauhtlatoatzina',
+    juliana_of_liege_virgin: 'Sv. Juliány z Liège, panny',  // TODO: `Juliana` or `Juliána`? … `Liège` or `Lutych`? (src: https://cs.wikipedia.org/wiki/Lutych) Note that in Slovak, `Lutych` is considered an archaic name for the city, while `Liège` is considered correct.
+    julie_billiart_virgin: 'Sv. Julie Billiart, panny',  // TODO: `Billiart` or `Billiartové`?
```
OK as is, both are possible

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    julie_billiart_virgin: 'Sv. Julie Billiart, panny',  // TODO: `Billiart` or `Billiartové`?
+    justin_martyr: 'Sv. Justina, mučedníka',
+    kateri_tekakwitha_virgin: 'Sv. Kateřiny Tekakwithy, panny',  // src: https://udalosti.signaly.cz/1304/svata-katerina-tekakwitha
+    katharine_drexel_virgin: 'Sv. Kateřiny Drexel, panny',  // TODO: `Drexel` or `Drexelové`?
```
both are possible

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    kentigern_of_scotland_bishop: 'Sv. Munga, biskupa',  // TODO: `Kentigern` or `Mungo`?
+    kevin_of_glendalough_abbot: 'Sv. Kevina, opata',
+    kieran_of_saigir_bishop: 'Sv. Kierána ze Saighir, biskupa',
+    kilian_of_wurzburg_bishop_and_companions_martyrs: 'Sv. Kiliána, biskupa, a druhů, mučedníků',  // TODO: `Kilian` or `Kilián`?
```
definitely Kilián

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    lawrence_of_brindisi_priest: 'Sv. Vavřince z Brindisi, kněze a učitele církve',
+    lawrence_of_rome_deacon: 'Sv. Vavřince, mučedníka',
+    lawrence_ruiz_and_companions_martyrs: 'Sv. Vavřince Ruize a druhů, mučedníků',
+    leander_of_seville_bishop: 'Sv. Leandra ze Sevilly, biskupa',  // TODO: `Sevilly` or `Seville`?
```
OK as is

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    paul_chen_changpin_and_companions_martyrs: 'Sv. Pavla Chen Changpin a druhů, mučedníků',
+    paulina_of_the_agonizing_heart_of_jesus_visintainer_virgin: 'Sv. Pavlíny od Bolestného Srdce Ježíšova Visintainer, panny',  // TODO: `Visintainer` or `Visintainerové`? `Pavlína` or `Paulína`? src: https://webcache.googleusercontent.com/search?q=cache:C4MW1p8DPd8J:dekanatprerov.cz/sites/default/files/slovo2012-11.pdf+&cd=1&hl=en&ct=clnk&gl=sk&client=firefox-b-d
+    paulinus_of_nola_bishop: 'Sv. Paulína Nolánského, biskupa',
+    paulinus_of_trier_bishop: 'Sv. Paulína z Trevíru, biskupa',  // TODO: `Trevírského` or `z Trevíru` or `z Trevíra`?
```
OK as is

igneus commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,797 @@
+    paulina_of_the_agonizing_heart_of_jesus_visintainer_virgin: 'Sv. Pavlíny od Bolestného Srdce Ježíšova Visintainer, panny',  // TODO: `Visintainer` or `Visintainerové`? `Pavlína` or `Paulína`? src: https://webcache.googleusercontent.com/search?q=cache:C4MW1p8DPd8J:dekanatprerov.cz/sites/default/files/slovo2012-11.pdf+&cd=1&hl=en&ct=clnk&gl=sk&client=firefox-b-d
+    paulinus_of_nola_bishop: 'Sv. Paulína Nolánského, biskupa',
+    paulinus_of_trier_bishop: 'Sv. Paulína z Trevíru, biskupa',  // TODO: `Trevírského` or `z Trevíru` or `z Trevíra`?
+    paulinus_of_york_bishop: 'Sv. Paulína z Yorku, biskupa',  // TODO: `Yorkského` or `z Yorku`?
```
OK as is

igneus reviewed (commented):
L344-555 reviewed. As always, uncommented TODOs mean "sorry, no idea" and no useful input is to be expected to them from me.

tukusejssirs commented:
@igneus, I have just finished localising the strings. There are still some strings to be reviewed which potentially contain errors. When you finish the review, I’ll complete the final steps (sort the keys and run `prettier`) and then, @emagnier, this PR will be mergable—of course, only after #213 is merged and this PR is rebased on top of it.

tukusejssirs commented on src/locales/cs.ts:
```diff
@@ -0,0 +1,796 @@
+    francis_of_assisi_patron_of_italy: 'Sv. Františka z Assisi, patrona Itálie',
+    adalbert_of_prague_bishop_patron_of_poland: 'Sv. Vojtěcha, biskupa, mučedníka a patrona Polska',
+    stanislaus_of_szczepanow_bishop_patron_of_poland: 'Sv. Stanislava, biskupa, mučedníka a patrona Polska',
+    wenceslaus_i_of_bohemia_martyr_patron_of_czech_nation: 'Sv. Václava, mučedníka, hlavního patrona českého národa',  // TODO: Add this to all other locales + czechRepublic calendar
```
What do you suggest, how should we translate `český národ`? If I recall correctly, you have once translated it as `the Czech nation`, however, [Wiki (en)](https://en.wikipedia.org/wiki/Czechs) uses `the Czech people`.

What do you think is better? :smiley:

**Update:** Note that whatever we choose here, will be reflected in other languages, for example:
- French: `la nation tchèque` vs `peuple tchèque`;
- Italian: `la nazione ceca` vs `popolo ceco`.

**Update 2:** Also I’d like to know how would I translate the patronage title into Latin: `patronus autem populus Bohemica` or `patronus de gente Bohemica` (both from Google Translate)?

**Update 3:** I have found the [transcript of homily of Benect XVI](http://www.vatican.va/content/benedict-xvi/en/homilies/2009/documents/hf_ben-xvi_hom_20090928_san-venceslao.html), which clearly says `Saint Wenceslaus, patron saint of the Czech nation`, therefore `nation` it is. :smiley: Good for me, that homily is translated to German, Spanish, French, Italian and Portuguese. :wink:

tukusejssirs replied:
Hey guys!

I need some proofing of you (just a single string :smiley:). Most of these should be correct _per se_.

Notes:
- @igneus: I’m quite confident the Latin translation is not grammatically correct. :wink:
- @igneus: I dropped the `hlavního` word because of how romcal works / will work, at least for now. Looking forward to your findings on the patronage matter in the Czech liturgical usage though. :smiley:

Assignments:
- @CodingAleCR: Spanish;
- @emagnier: French;
- @igneus: Latin;
- @gtortone: Italian;
- @jarosz: Polish;
- @HenriqueDerosa: Brazilian Portuguese.

```js
// Original: Czech
wenceslaus_i_of_bohemia_martyr_patron_of_czech_nation:
  'Sv. Václava, mučedníka a patrona českého národa',

en : 'Saint Wenceslaus, Martyr and patron of the Czech nation',
es : 'San Wenceslao de Bohemia, Mártir y Patrono de la nación checa',  // TODO: proof; also add a key for the celebration without patronage title
fr : 'Saint Venceslas, Martyr et Patron de la nation tchèque (✝ 929)',
it : 'San Venceslao, martire e patrono della nazione ceca',  // TODO: proof
la : 'S. Wenceslai, martyris et patroni de gente Bohemica',  // TODO: proof
pl : 'Św. Wacława, męczennika i patrona narodu czeskiego',  // TODO: proof
pt-br : 'São Venceslau, mártir e patrono da nação tcheca',  // TODO: proof
sk : 'Svätého Václava, mučeníka a patróna českého národa',
```

**PS—Thanks in advance! :wink:**

emagnier replied:
`fr : 'Saint Venceslas, Martyr et Patron de la Tchéquie (✝ 929)'`

tukusejssirs replied:
@emagnier, thanks for your comment, however, your localisation is not quite correct. I had a conversation about this with @igneus (a native Czech). I too thought that `the Czech nation` equals the Czech Republic / Czechia, however, it is not so. The difference is subtle, but it does exist.

Refer please to this [homily of Benedict XVI](https://www.vatican.va/content/benedict-xvi/fr/homilies/2009/documents/hf_ben-xvi_hom_20090928_san-venceslao.html). In the heading, you can find `Saint Venceslas, patron de la nation`. I simply added the `tchèque` word to distinguish of which nation he is a patron.

emagnier replied:
Thanks for the clarifications. We are less used to hearing this in this form in French, but I understand the difference. So keep what you have suggested initially.

tukusejssirs replied:
> We are less used to hearing this in this form in French

So am I and I think most Slovaks too. :smiley:

Anyway, thanks for the proofing. :wink:

CodingAleCR replied:
Hi, the Spanish translation looks good to me. Cheers!
Although you could also use "Patrono de la República Checa" instead of "... de la nación checa" as it's usually associated with the country (for example in [this article](https://www.aciprensa.com/santos/santo.php?id=290) by ACIPrensa). Either of those works so no need to change if you don't feel like it is needed.

tukusejssirs replied:
I suggest we should not wait any longer for others to approve my translations. I am fairly confident that they are all more or less correct.

tukusejssirs commented:
@igneus, were could I get the Czech National Proper? I know that you list the celebrations in [`czech-cs.txt`](https://github.com/igneus/calendarium-romanum/blob/master/data/czech-cs.txt) file, however, it contains celebrations that come from the GRC. I could get through each of them, however, I wish I could access the Proper with the following information:
- date, rank, celebration name of inserted celebrations;
- date and name of moved celebrations (in comparison with the GRC);
- dropped celebrations if any (in comparison with the GRC);
- raised/lowered celebrations (in comparison with the GRC);
- any other possible changes in comparison with the GRC (e.g. added titles to celebrated persons).

Thanks in advance!

tukusejssirs committed:
Rebase on top of `_F-TypeScript` branch

tukusejssirs commented on src/calendars/czechRepublic.ts:
```diff
@@ -21,14 +25,28 @@ const dates = async (config: RomcalConfig): Promise<Array<LiturgicalDayInput>> =
+    // {
+    //   key: 'adalbert_of_prague_bishop',
+    //   rank: Ranks.MEMORIAL,
+    //   date: dayjs.utc(`${year}-4-23`),
```
@igneus, is St Adalbert celebrated as `[principal] patron of the Archdiocese of Prague` in whole Czech Republic (as it is listed in [`czech-cs.txt` (L62)](https://github.com/igneus/calendarium-romanum/blob/4918754ca64d501e3c62165d1512adbbabcc9148/data/czech-cs.txt#L62)) or is it celebrated as a memorial (as it commented above in the romcal codebase) or is it celebrated as a mere optional memorial (inherited from the GRC)?

tukusejssirs committed:
Pre-final tidy up

- remove all comments of mine from `cs.ts`;
- sort keys in `celebrations` and `sanctoral`;
- run `prettier`.

tukusejssirs commented on src/calendars/czechRepublic.ts:
```diff
@@ -81,12 +109,9 @@ const dates = async (config: RomcalConfig): Promise<Array<LiturgicalDayInput>> =
     {
       key: 'cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_copatrons_of_europe',
-      rank: Ranks.SOLEMNITY,
+      rank: Ranks.SOLEMNITY, // TODO: Should we add `Patrons of Moravia`?
```
@igneus, I am not sure if it should be celebrated with patronage title of `Copatrons of Europe` or `Copatrons of Europe and Patrons of Moravia` (as it is listed in [`czech-cs.txt`](https://github.com/igneus/calendarium-romanum/blob/4918754ca64d501e3c62165d1512adbbabcc9148/data/czech-cs.txt#L122)). I remind you I need the celebration metadata as it is celebrated in the particular calendar for Czech Republic, not for Dioceses nor Bohemia nor Moravia.

tukusejssirs reviewed (commented):
This PR is mergable as is, however, it’d be nice if the following things are reviewed before merging:

- [ ] @igneus: proof the Czech National Proper (see [Update 3 in this comment](https://github.com/romcal/romcal/pull/248#issuecomment-722358775) for the Proper I compiled):
   - there are two new review comments that are connected to this:
      - regarding the patronage titles of St Adalbert (see [this comment](https://github.com/romcal/romcal/pull/248#discussion_r537119842));
      - regarding the patronage titles of Sts Cyril and Methodius (see [this comment](https://github.com/romcal/romcal/pull/248#discussion_r537885558));
- [ ] @igneus: check the inflection of `Jana Marie Vianneye` (see [this comment](https://github.com/romcal/romcal/pull/248/files#r516877800));
- [ ] @igneus: check the inflection of `Josemaría Escrivá de Balaguer` (see [this comment](https://github.com/romcal/romcal/pull/248/files#r516887877));
- [ ] @emagnier / @igneus / anyone: [just for my information] what is the rank of All souls?
   - In `czech-cs.txt`, you list it as solemnity (`s`);
   - [wiki (en)](https://en.wikipedia.org/wiki/General_Roman_Calendar#November), they say it is `ranked with solemnities`;
   - in the Slovak _Ordos_, we don’t list any rank with the celebration.
- [ ] I’d be nice if the localisations in [this comment](https://github.com/romcal/romcal/pull/248#discussion_r518015314) would be proofed by native speakers (@CodingAleCR, @gtortone, @igneus, @jarosz, @HenriqueDoresa).

**Update:** These (potential) issues are moved to separate issue (#267), except for the last one regarding the lacalisation proofing (for which we should wait no more).

tukusejssirs committed:
Fix some issues from missed review comments

igneus commented:
Hi, I just want to announce that I'm not going to react to the numerous mentions any soon (and possibly not at all). My involvement in romcal's development has a very low priority among my tasks and currently I'm simply not going to do anything here, as I have other stuff to do. Sorry. Consider looking for some other "native Czech" to assist you. Or perhaps maintain a single long list of "questions for igneus" and from time to time (when I have the time and feel like contributing a bit to romcal) I'll eventually come answer them.

tukusejssirs committed:
Fix a small issue in `pl.ts`

tukusejssirs commented:
@emagnier, by creating the #267 issue I am unblocking this PR. Therefore this PR is mergable.
